db/migrate/20220720214646_add_delete_cascade_from_orgs_down.rb

```diff
@@ -0,0 +1,7 @@
+class AddDeleteCascadeFromOrgsDown < ActiveRecord::Migration[7.0]
+  def change
+    remove_foreign_key :schemes, :organisations, column: "owning_organisation_id"
+    remove_foreign_key :schemes, :organisations, column: "managing_organisation_id"
```

Comment:
I don't think we want to remove the managing org foreign key

Reply:
Fixed in latest commit